### PR TITLE
BigQuery: More reliable insert-into-select 

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
Not sure how this issue wasn't caught until now, but not explicitly specifying the column order in the QRep insert into select statement can cause it to fail due to column-value insert mismatch (ex: expected int but got bool for _peerdb_is_deleted)

### CODE REVIEW

File: flow/connectors/bigquery/qrep_avro_sync.go [hunk 1]
```diff
@@ -184,18 +190,18 @@ func (s *QRepAvroSyncMethod) SyncQRepRecords(
 	)
 	bqClient := s.connector.client
 
+	insertColumns := make([]string, 0, len(dstTableMetadata.Schema))
+	for _, col := range dstTableMetadata.Schema {
+		insertColumns = append(insertColumns, fmt.Sprintf("`%s`", col.Name))
+	}
```
Comment (serprex) [inline flow/connectors/bigquery/qrep_avro_sync.go:196]: Need to include synced_at/deleted columns

**Comment by iskakaushik** [comment]
is there a way to add a test for this?

File: flow/connectors/bigquery/qrep_avro_sync.go (hunk 1 above)
Comment (Amogh-Bharadwaj) [reply flow/connectors/bigquery/qrep_avro_sync.go:196]: for example that's how we're setting their values in getTransformedColumns

**Comment by iskakaushik** [comment]
@Amogh-Bharadwaj  isn't able to reproduce this issue, @serprex do you think we can grant a test exemption here given that this PR qualifies the statements and makes them more robust?

**Comment by serprex** [comment]
> avro_transform_test.go:37: Transform SQL is not correct. Got: [ST_GEOGFROMTEXT(`col1`) AS `col1` PARSE_JSON(`col2`,wide_number_mode=>'round') AS `col2` `camelCol4` CURRENT_TIMESTAMP AS `sync_col`]

If tests are passing then yes, I'd consider this code tested since it's part of our existing test coverage

**Comment by iskakaushik** [comment]
needed for customer.

**Pull request merged**